R/draws.R

```diff
@@ -50,10 +50,14 @@
 #' MMRM parameter estimates from the original dataset and from each leave-one-subject-out sample.
 #' This method can be specified by using `method = method_condmean()` with
 #' argument `type = "jackknife"`.
+#' - Bootstrapped Maximum Likelihood MI: `draws` returns the MMRM parameter estimates from
+#' a given number of bootstrap samples needed to perform random imputations of the bootstrapped samples.
+#' This method can be specified by using `method = method_bmlmi()`.
```

Comment:
This description doesn't feel that clear to me (maybe from the double use of "bootstrapped").  Will try and re-word later but I think we should change it somehow

Reply:
mmm, I agree but at the same time if I think of it I kind of understand that I need to specify twice "bootstrapped samples".. I am happy to change it however.